### DISCUSSION TITLE
SMTPS is obsolete, RFC8314 says clearly: STARTTLS

### DISCUSSION THREAD

**Discussion by ser** [Mailu/Mailu#3371] [General]
Accordingly to https://www.rfc-editor.org/rfc/rfc8314#section-3.3 SMTPS is obsolete and should be replaced by STARTTLS. Why mailu is still displaying SMTPS as the default route for sending mail for clients?

**Comment by ghostwheel42** [comment]
RFC8314 is about "Cleartext Considered Obsolete", not "SMTPS".
Section 3.3 states that clients and servers SHOULD implement both STARTTLS and Implicit TLS for message submission.
For pop3, imap, submission we chose to not do STARTTLS by default as it has security implications, but you can re-enable it if you like. See https://mailu.io/2024.06/releases.html#starttls-ports-disabled-by-default

**Reply by ser** [reply]
OK, I didn't properly understand the meaning of the RFC. Thanks for explanation.
